locations/spiders/ruler_foods_us.py

```diff
@@ -1,22 +1,28 @@
-import re
+from typing import Iterable
 
-from locations.hours import DAYS, OpeningHours
-from locations.storefinders.store_locator_plus_self import StoreLocatorPlusSelfSpider
-from locations.structured_data_spider import clean_facebook
+import scrapy
+from requests import Response
+from scrapy.http import JsonRequest
 
+from locations.dict_parser import DictParser
+from locations.items import Feature
+from locations.user_agents import BROWSER_DEFAULT
 
-class RulerFoodsUSSpider(StoreLocatorPlusSelfSpider):
+
+class RulerFoodsUSSpider(scrapy.Spider):
     name = "ruler_foods_us"
     item_attributes = {"brand": "Ruler Foods", "brand_wikidata": "Q17125470"}
-    allowed_domains = ["rulerfoods.com"]
-    iseadgg_countries_list = ["US"]
-    search_radius = 500
-    max_results = 50
+    allowed_domains = ["falcon.shop.inmar.io"]
+    custom_settings = {"USER_AGENT": BROWSER_DEFAULT}
+
+    async def start(self):
```

Comment:
I think the first call should be the equivalent of `curl 'https://falcon.shop.inmar.io/v2/session' -X POST -H 'inmar-banner-id: ruler_foods'` using JsonRequest then parse the `sessionId` from this response to pass to a second JsonRequest for store locations.